src/DigitalTwin/CodeGeneratorCore.ts

```diff
@@ -533,7 +533,8 @@ export class CodeGeneratorCore {
     context: vscode.ExtensionContext,
     channel: vscode.OutputChannel
   ): Promise<CodeGeneratorConfigItem | null> {
-    const extensionPackage = require(context.asAbsolutePath("./package.json"));
+    //const extensionPackage = require(context.asAbsolutePath("./package.json"));
+    const extensionPackage = JSON.parse(fs.readFileSync(context.asAbsolutePath("./package.json"), "utf8"));

```

Comment:
I think you could use getJsonContent to get JSON content, which is in src\DigitalTwin\pnp\src\common\utility.ts

Reply:
OK